src/Player.cpp

```diff
@@ -993,6 +993,58 @@ void Player::Update( float fDeltaTime )
 		}
 	}
 
+	// track held misses
+	{
+		float largestWindow = .0;
```

Comment:
(random nit so feel free to ignore). Can write `0.0f` to ensure tell the compiler it's not double precision (which is the default).

Reply:
I don't have a strong opinion on the formatting, so I changed it to `0.0f`. That's seems to be the style that's more prevalent in the SM codebase anyway.